### PR TITLE
[Group 3] Adding lane detection and road properties handler

### PR DESCRIPTION
**Pull request by vishakhsuresh98**
This PR adds the road driving group's lane detector and files for launching and testing our detection components.

It also includes a program to find the properties (eg: name, speed limit, number of lanes) of roads using the latitude and longitude.

### CODE REVIEW

File: GEMstack/knowledge/defaults/computation_graph.yaml
Comment (krishauser) [inline GEMstack/knowledge/defaults/computation_graph.yaml:1]: Why not just keep this as vehicle_lane?

Comment (krishauser) [reply GEMstack/knowledge/defaults/computation_graph.yaml:1]: I suggest you make this a “roadgraph_update” component that outputs a roadgraph.

**Review by krishauser — CHANGES_REQUESTED** [review]
Suggest changing the launch to call this a roadgraph_update component

File: GEMstack/knowledge/defaults/computation_graph.yaml
Comment (vishakhsuresh98) [reply GEMstack/knowledge/defaults/computation_graph.yaml:1]: - Made the `LaneDetector` a `roadgraph_update` component
- Updated the launch file
- Restored the computation graph and the `AllState` class to their previous states

**Pull request merged**